samples/sqltransport/store-and-forward/sample.md

```diff
@@ -0,0 +1,101 @@
+---
+title: SQL Server transport store-and-forward
+summary: How to add store-and-forward functionality for external-facing endpoints
+reviewed: 2016-03-21
+tags:
+- SQL Server
+- Store-and-forward
+related:
+- nservicebus/outbox
+- nservicebus/sqlserver
+redirects:
+- samples/sqltransport-outbox-store-and-forward
+- samples/outbox/store-and-forward
+---
+
+
+## Prerequisites
+
+ 1. Ensure an instance SQL Server Express installed and accessible as `.\SQLEXPRESS`. Create two databases: `sender` and `receiver`.
+
+
+## Running the project
+
+ 1. Start the Sender project (right-click on the project, select the `Debug > Start new instance` option).
+ 1. Start the Receiver project. 
```

Comment:
when people DL the sample zip the correct startup projects will be set. so do we really need to tell them to start both ?